diff -urN aaa/pango-1.1.4/examples/Makefile.am pango-1.1.4/examples/Makefile.am
--- aaa/pango-1.1.4/examples/Makefile.am	Tue Nov 26 14:36:40 2002
+++ pango-1.1.4/examples/Makefile.am	Tue Dec  3 21:25:17 2002
@@ -6,9 +6,9 @@
 	-DG_DISABLE_DEPRECATED		\
 	-I$(top_srcdir)			\
 	$(QT_INCLUDES)			\
+	$(FREETYPE_CFLAGS)	\
 	$(GLIB_CFLAGS)			\
-	$(FONTCONFIG_CFLAGS)		\
-	$(FREETYPE_CFLAGS)
+	$(FONTCONFIG_CFLAGS)
 
 ###################################################
 
