diff -urN pango-1.16.0/configure.in pango-1.16.0.new/configure.in
--- pango-1.16.0/configure.in	2007-02-26 20:13:10.000000000 +0100
+++ pango-1.16.0.new/configure.in	2007-03-10 14:33:28.000000000 +0100
@@ -282,6 +282,10 @@
 AC_SUBST(WIN32_LIBS)
 AM_CONDITIONAL(HAVE_WIN32, $have_win32) 
 
+ARCH=${CC%%-*}
+AC_SUBST(ARCH)
+AC_DEFINE_UNQUOTED(ARCH, "${ARCH}", [Architecture])
+
 # Ensure MSVC-compatible struct packing convention is used when
 # compiling for Win32 with gcc.
 # What flag to depends on gcc version: gcc3 uses "-mms-bitfields", while
diff -urN pango-1.16.0/modules/Makefile.am pango-1.16.0.new/modules/Makefile.am
--- pango-1.16.0/modules/Makefile.am	2007-01-03 09:15:03.000000000 +0100
+++ pango-1.16.0.new/modules/Makefile.am	2007-03-10 14:35:45.000000000 +0100
@@ -28,15 +28,15 @@
 
 install-data-local: $(top_builddir)/pango/pango-querymodules
 	@if $(RUN_QUERY_MODULES_TEST) ; then 				\
-	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango \&\& 	\
+	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango-$(ARCH) \&\& 	\
 	  $(top_builddir)/pango/pango-querymodules 			\
-		\> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;	\
-	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango && 		\
+		\> $(DESTDIR)$(sysconfdir)/pango-$(ARCH)/pango.modules ;	\
+	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango-$(ARCH) && 		\
 	  $(top_builddir)/pango/pango-querymodules 			\
-		> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;		\
+		> $(DESTDIR)$(sysconfdir)/pango-$(ARCH)/pango.modules ;		\
 	else								\
 	  echo "***" ; 							\
-	  echo "*** Warning: $(sysconfdir)/pango/pango.modules" ; 	\
+	  echo "*** Warning: $(sysconfdir)/pango-$(ARCH)/pango.modules" ; 	\
 	  echo "*** not created" ; 					\
 	  echo "*** Generate this file on the target system" ; 		\
 	  echo "*** using pango-querymodules" ; 			\
@@ -44,7 +44,7 @@
 	fi
 
 uninstall-local:
-	$(RM) $(DESTDIR)$(sysconfdir)/pango/pango.modules
+	$(RM) $(DESTDIR)$(sysconfdir)/pango-$(ARCH)/pango.modules
 
 pango.modules:						\
 	$(SUBDIRS)					\
diff -urN pango-1.16.0/pango/Makefile.am pango-1.16.0.new/pango/Makefile.am
--- pango-1.16.0/pango/Makefile.am	2007-02-18 18:10:43.000000000 +0100
+++ pango-1.16.0.new/pango/Makefile.am	2007-03-10 14:53:50.000000000 +0100
@@ -158,7 +158,7 @@
 if HAVE_X
 pangoinclude_HEADERS += pangox.h
 lib_LTLIBRARIES += libpangox-1.0.la
-aliasdir = $(sysconfdir)/pango
+aliasdir = $(sysconfdir)/pango-$(ARCH)
 alias_DATA = pangox.aliases
 endif
 
diff -urN pango-1.16.0/pango/pango-utils.c pango-1.16.0.new/pango/pango-utils.c
--- pango-1.16.0/pango/pango-utils.c	2007-01-23 00:41:43.000000000 +0100
+++ pango-1.16.0.new/pango/pango-utils.c	2007-03-10 14:37:03.000000000 +0100
@@ -690,7 +690,7 @@
 
   return result;
 #else
-  return SYSCONFDIR "/pango";
+  return SYSCONFDIR "/pango-" ARCH;
 #endif
 }
 
diff -urN pango-1.16.0/pango/pangox-fontmap.c pango-1.16.0.new/pango/pangox-fontmap.c
--- pango-1.16.0/pango/pangox-fontmap.c	2007-01-23 00:41:43.000000000 +0100
+++ pango-1.16.0.new/pango/pangox-fontmap.c	2007-03-10 14:38:04.000000000 +0100
@@ -921,7 +921,7 @@
   int n;
 
   if (!files_str)
-    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pango/pangox.aliases");
+    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pango-" ARCH "/pangox.aliases");
 
   files = pango_split_file_list (files_str);
 
