--- pango-1.12.4/modules/Makefile.am	2006-02-13 21:03:21.000000000 +0200
+++ pango-1.12.4.archdir/modules/Makefile.am	2007-10-22 22:05:23.774815525 +0300
@@ -28,15 +28,15 @@
 
 install-data-local: $(top_builddir)/pango/pango-querymodules
 	@if $(RUN_QUERY_MODULES_TEST) ; then 				\
-	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango \&\& 	\
+	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir) \&\& 	\
 	  $(top_builddir)/pango/pango-querymodules 			\
-		\> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;	\
-	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango && 		\
+		\> $(DESTDIR)$(sysconfdir)/pango.modules ;	\
+	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir) && 		\
 	  $(top_builddir)/pango/pango-querymodules 			\
-		> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;		\
+		> $(DESTDIR)$(sysconfdir)/pango.modules ;		\
 	else								\
 	  echo "***" ; 							\
-	  echo "*** Warning: $(sysconfdir)/pango/pango.modules" ; 	\
+	  echo "*** Warning: $(sysconfdir)/pango.modules" ; 	\
 	  echo "*** not created" ; 					\
 	  echo "*** Generate this file on the target system" ; 		\
 	  echo "*** using pango-querymodules" ; 			\
@@ -44,7 +44,7 @@
 	fi
 
 uninstall-local:
-	$(RM) $(DESTDIR)$(sysconfdir)/pango/pango.modules
+	$(RM) $(DESTDIR)$(sysconfdir)/pango.modules
 
 pango.modules: $(top_builddir)/pango/pango-querymodules $(SUBDIRS)
 	@if find . -name '*.so' | grep 'so' > /dev/null ; then \
--- pango-1.12.4/pango/Makefile.am	2006-07-25 04:04:30.000000000 +0300
+++ pango-1.12.4.archdir/pango/Makefile.am	2007-10-22 22:04:51.514077302 +0300
@@ -148,7 +148,7 @@
 if HAVE_X
 pangoinclude_HEADERS += pangox.h
 lib_LTLIBRARIES += libpangox-1.0.la
-aliasdir = $(sysconfdir)/pango
+aliasdir = $(sysconfdir)
 alias_DATA = pangox.aliases
 endif
 
--- pango-1.12.4/pango/pango-utils.c	2006-05-22 07:30:18.000000000 +0300
+++ pango-1.12.4.archdir/pango/pango-utils.c	2007-10-22 22:04:51.514077302 +0300
@@ -606,7 +606,7 @@
 
   return result;
 #else
-  return SYSCONFDIR "/pango";
+  return SYSCONFDIR;
 #endif
 }
 
--- pango-1.12.4/pango/pangox-fontmap.c	2006-02-06 01:48:27.000000000 +0200
+++ pango-1.12.4.archdir/pango/pangox-fontmap.c	2007-10-22 22:04:51.514077302 +0300
@@ -922,7 +922,7 @@
   int n;
 
   if (!files_str)
-    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pango/pangox.aliases");
+    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pangox.aliases");
 
   files = pango_split_file_list (files_str);
   
