diff -uNr pango-1.30.1/modules/Makefile.am pango-1.30.1.new/modules/Makefile.am
--- pango-1.30.1/modules/Makefile.am	2011-08-16 03:11:08.000000000 +0200
+++ pango-1.30.1.new/modules/Makefile.am	2012-06-11 16:30:03.042700937 +0200
@@ -28,15 +28,15 @@
 
 install-data-local: $(top_builddir)/pango/pango-querymodules$(EXEEXT)
 	@if $(RUN_QUERY_MODULES_TEST) && test -z "$(DESTDIR)" ; then 	\
-	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango \&\& 	\
+	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir) \&\& 	\
 	  $(top_builddir)/pango/pango-querymodules$(EXEEXT) 		\
-		\> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;	\
-	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango && 		\
+		\> $(DESTDIR)$(sysconfdir)/pango.modules ;	\
+	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir) && 		\
 	  $(top_builddir)/pango/pango-querymodules$(EXEEXT) 		\
-		> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;		\
+		> $(DESTDIR)$(sysconfdir)/pango.modules ;		\
 	else								\
 	  echo "***" ; 							\
-	  echo "*** Warning: $(sysconfdir)/pango/pango.modules" ; 	\
+	  echo "*** Warning: $(sysconfdir)/pango.modules" ; 	\
 	  echo "*** not created" ; 					\
 	  echo "*** Generate this file on the target system" ; 		\
 	  echo "*** using pango-querymodules" ; 			\
@@ -44,7 +44,7 @@
 	fi
 
 uninstall-local:
-	$(RM) $(DESTDIR)$(sysconfdir)/pango/pango.modules
+	$(RM) $(DESTDIR)$(sysconfdir)/pango.modules
 
 pango.modules:						\
 	$(SUBDIRS)					\
diff -uNr pango-1.30.1/pango/pango-utils.c pango-1.30.1.new/pango/pango-utils.c
--- pango-1.30.1/pango/pango-utils.c	2012-06-02 04:43:05.000000000 +0200
+++ pango-1.30.1.new/pango/pango-utils.c	2012-06-11 16:30:03.046034427 +0200
@@ -740,7 +740,7 @@
       if (sysconfdir != NULL)
 	tpm_result = g_build_filename (sysconfdir, "pango", NULL);
       else
-	tmp_result = SYSCONFDIR "/pango";
+	tmp_result = SYSCONFDIR;
 #endif
     }
   return result;
