diff -Nru pango-1.1.1.orig/pango/pangoxft.h pango-1.1.1/pango/pangoxft.h
--- pango-1.1.1.orig/pango/pangoxft.h	Tue Aug  6 03:54:38 2002
+++ pango-1.1.1/pango/pangoxft.h	Fri Oct  4 14:05:21 2002
@@ -31,7 +31,7 @@
 #define _XFT_NO_COMPAT
 #define _XFTCOMPAT_H_
 #include <X11/Xlib.h>
-#include <X11/Xft/Xft.h>
+#include <X11/Xft2/Xft.h>
 #if defined(XftVersion) && XftVersion >= 20000
 #else
 #error "must have Xft version 2 or newer"
