diff -Nur pango-1.10.2/configure.in pango-1.10.2.arch_confdir/configure.in
--- pango-1.10.2/configure.in	2005-11-29 05:28:16.000000000 +0000
+++ pango-1.10.2.arch_confdir/configure.in	2005-12-10 16:57:38.000000000 +0000
@@ -270,6 +270,10 @@
 AC_SUBST(WIN32_LIBS)
 AM_CONDITIONAL(HAVE_WIN32, $have_win32) 
 
+ARCH=${CC%%-*}
+AC_SUBST(ARCH)
+AC_DEFINE_UNQUOTED(ARCH, "${ARCH}", [Architecture])
+
 #
 # Checks for Cairo
 #
diff -Nur pango-1.10.2/modules/Makefile.am pango-1.10.2.arch_confdir/modules/Makefile.am
--- pango-1.10.2/modules/Makefile.am	2005-06-21 16:02:55.000000000 +0000
+++ pango-1.10.2.arch_confdir/modules/Makefile.am	2005-12-10 16:57:38.000000000 +0000
@@ -28,15 +28,15 @@
 
 install-data-local: $(top_builddir)/pango/pango-querymodules
 	@if $(RUN_QUERY_MODULES_TEST) ; then 				\
-	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango \&\& 	\
+	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango-$(ARCH) \&\& 	\
 	  $(top_builddir)/pango/pango-querymodules 			\
-		\> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;	\
-	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango && 		\
+		\> $(DESTDIR)$(sysconfdir)/pango-$(ARCH)/pango.modules ;	\
+	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango-$(ARCH) && 		\
 	  $(top_builddir)/pango/pango-querymodules 			\
-		> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;		\
+		> $(DESTDIR)$(sysconfdir)/pango-$(ARCH)/pango.modules ;		\
 	else								\
 	  echo "***" ; 							\
-	  echo "*** Warning: $(sysconfdir)/pango/pango.modules" ; 	\
+	  echo "*** Warning: $(sysconfdir)/pango-$(ARCH)/pango.modules" ; 	\
 	  echo "*** not created" ; 					\
 	  echo "*** Generate this file on the target system" ; 		\
 	  echo "*** using pango-querymodules" ; 			\
diff -Nur pango-1.10.2/pango/pango-utils.c pango-1.10.2.arch_confdir/pango/pango-utils.c
--- pango-1.10.2/pango/pango-utils.c	2005-11-23 17:53:30.000000000 +0000
+++ pango-1.10.2.arch_confdir/pango/pango-utils.c	2005-12-10 16:57:38.000000000 +0000
@@ -672,7 +672,7 @@
 
   return result;
 #else
-  return SYSCONFDIR "/pango";
+  return SYSCONFDIR "/pango-" ARCH ;
 #endif
 }
 
diff -Nur pango-1.10.2/pango/pangox-fontmap.c pango-1.10.2.arch_confdir/pango/pangox-fontmap.c
--- pango-1.10.2/pango/pangox-fontmap.c	2005-07-22 19:05:45.000000000 +0000
+++ pango-1.10.2.arch_confdir/pango/pangox-fontmap.c	2005-12-10 16:58:49.000000000 +0000
@@ -30,6 +30,8 @@
 /* For XExtSetCloseDisplay */
 #include <X11/Xlibint.h>
 
+#include "config.h"
+
 #include "pango-engine-private.h"
 #include "pango-fontmap.h"
 #include "pango-utils.h"
@@ -917,7 +919,7 @@
   int n;
 
   if (!files_str)
-    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pango/pangox.aliases");
+    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pango-" ARCH "/pangox.aliases");
 
   files = pango_split_file_list (files_str);
   
--- pango-1.10.2/pango/Makefile.am	2006-02-03 06:42:33.000000000 +0000
+++ pango-1.10.2.arch_confdir/pango/Makefile.am	2006-06-27 14:28:44.000000000 +0000
@@ -148,7 +148,7 @@
 if HAVE_X
 pangoinclude_HEADERS += pangox.h
 lib_LTLIBRARIES += libpangox-1.0.la
-aliasdir = $(sysconfdir)/pango
+aliasdir = $(sysconfdir)/pango-$(ARCH)
 alias_DATA = pangox.aliases
 endif
 
