--- pango-1.20.0/configure.in.orig	2008-03-10 16:19:53.000000000 +0100
+++ pango-1.20.0/configure.in	2008-03-12 23:58:25.478798103 +0100
@@ -124,6 +124,8 @@
 fi
 
 AM_PROG_LIBTOOL
+dnl when using libtool 2.x create libtool early, because it's used in configure
+m4_ifdef([LT_OUTPUT], [LT_OUTPUT])
 
 AC_MSG_CHECKING([for some Win32 platform])
 case "$host" in
@@ -664,7 +664,7 @@
 case $enable_explicit_deps in
   auto)
     export SED
-    deplibs_check_method=`(./libtool --config; echo 'eval echo $deplibs_check_method') | sh`
+    deplibs_check_method=$( (./libtool --config; echo 'eval echo \"$deplibs_check_method\"') | sh)
     if test "x$deplibs_check_method" '!=' xpass_all || test "x$enable_static" = xyes ; then
       enable_explicit_deps=yes  
     else
