--- pango-1.18.0/modules/Makefile.am.orig	2007-07-05 23:30:28.000000000 +0200
+++ pango-1.18.0/modules/Makefile.am	2007-08-26 20:39:17.982104313 +0200
@@ -28,15 +28,15 @@
 
 install-data-local: $(top_builddir)/pango/pango-querymodules$(EXEEXT)
 	@if $(RUN_QUERY_MODULES_TEST) && test -z "$(DESTDIR)" ; then 	\
-	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango \&\& 	\
+	  echo $(mkinstalldirs) $(DESTDIR)$(sysconfdir) \&\& 	\
 	  $(top_builddir)/pango/pango-querymodules$(EXEEXT) 		\
-		\> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;	\
-	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir)/pango && 		\
+		\> $(DESTDIR)$(sysconfdir)/pango.modules ;	\
+	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir) && 		\
 	  $(top_builddir)/pango/pango-querymodules$(EXEEXT) 		\
-		> $(DESTDIR)$(sysconfdir)/pango/pango.modules ;		\
+		> $(DESTDIR)$(sysconfdir)/pango.modules ;		\
 	else								\
 	  echo "***" ; 							\
-	  echo "*** Warning: $(sysconfdir)/pango/pango.modules" ; 	\
+	  echo "*** Warning: $(sysconfdir)/pango.modules" ; 	\
 	  echo "*** not created" ; 					\
 	  echo "*** Generate this file on the target system" ; 		\
 	  echo "*** using pango-querymodules" ; 			\
@@ -44,7 +44,7 @@
 	fi
 
 uninstall-local:
-	$(RM) $(DESTDIR)$(sysconfdir)/pango/pango.modules
+	$(RM) $(DESTDIR)$(sysconfdir)/pango.modules
 
 pango.modules:						\
 	$(SUBDIRS)					\
--- pango-1.16.2.orig/pango/Makefile.am	2007-04-12 23:54:04.006011079 +0200
+++ pango-1.16.2/pango/Makefile.am	2007-04-12 23:54:27.631054049 +0200
@@ -158,7 +158,7 @@
 if HAVE_X
 pangoinclude_HEADERS += pangox.h
 lib_LTLIBRARIES += libpangox-1.0.la
-aliasdir = $(sysconfdir)/pango
+aliasdir = $(sysconfdir)
 alias_DATA = pangox.aliases
 endif
 
--- pango-1.16.2.orig/pango/pango-utils.c	2007-04-12 23:54:03.899351741 +0200
+++ pango-1.16.2/pango/pango-utils.c	2007-04-12 23:54:27.631054049 +0200
@@ -690,7 +690,7 @@
 
   return result;
 #else
-  return SYSCONFDIR "/pango";
+  return SYSCONFDIR;
 #endif
 }
 
--- pango-1.16.2.orig/pango/pangox-fontmap.c	2007-04-12 23:54:04.026009703 +0200
+++ pango-1.16.2/pango/pangox-fontmap.c	2007-04-12 23:54:27.634387153 +0200
@@ -921,7 +921,7 @@
   int n;
 
   if (!files_str)
-    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pango/pangox.aliases");
+    files_str = g_strdup ("~/.pangox_aliases:" SYSCONFDIR "/pangox.aliases");
 
   files = pango_split_file_list (files_str);
 
