diff -urN aaa/pango-1.2.4/modules/indic/Makefile.am pango-1.2.4/modules/indic/Makefile.am
--- aaa/pango-1.2.4/modules/indic/Makefile.am	2003-08-08 19:04:59.000000000 +0200
+++ pango-1.2.4/modules/indic/Makefile.am	2003-08-27 01:39:01.000000000 +0200
@@ -12,7 +12,9 @@
 	indic-xft.c		\
 	indic-ot-class-tables.c	\
 	indic-ot.c		\
-	indic-ot.h
+	indic-ot.h		\
+	mprefixups.c		\
+	mprefixups.h
 
 pangolibs = $(top_builddir)/pango/libpango-$(PANGO_API_VERSION).la $(GLIB_LIBS)
 pangoxlibs = $(top_builddir)/pango/libpangox-$(PANGO_API_VERSION).la $(X_LIBS) $(pangolibs)
